account_banking_ach_direct_debit_portal/controllers/autopay_rules_controller.py

```diff
@@ -0,0 +1,48 @@
+import json
+from datetime import date
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class InvoiceController(CustomerPortal):
```

Comment:
2 things:
1) by convention, name of the file should not be suffixed by `_controller.py`
2) the name of the controller should match its filename